templates/projects/js/annotation_guidelines_button.js

```diff
@@ -0,0 +1,10 @@
+
+<script>
+$(".collapse-guidelines-button").on('click', function(){
+    var showInstructionsMsg = 'Show Instructions';
+    var hideInstructionsMsg = 'Hide Instructions';
+    $(this).text(function(i, v){
+        return v === showInstructionsMsg ? hideInstructionsMsg : showInstructionsMsg
```

Comment:
Similar comment to the above. :)

```js
$(this).text((i, v) => {
  return `${v.startsWith('Show') ? 'Hide' : 'Show'} Instructions`;
});
```

Reply:
I  apply a similar change as you suggested but Im not using template since they are not supported on older browsers. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Ok, I understand. Btw, is there a specific version of Chrome that we're supporting from? Chrome typically auto-updates, in which case templating should be supported, but if some users are still on an older version than this is certainly a valid use-case. Will post in chat. Thank you. :)

Reply:
I don't think there is a specific version. Also we don't know if the workers updates their browser or not.

Reply:
Hm, we should be able to gather this from analytics.